CHANGELOG.md

```diff
@@ -124,6 +124,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (x/auth/vesting) [#13502](https://github.com/cosmos/cosmos-sdk/pull/13502) Add Amino Msg registration for `MsgCreatePeriodicVestingAccount`.
 * (x/auth)[#13780](https://github.com/cosmos/cosmos-sdk/pull/13780) `id` (type of int64) in `AccountAddressByID` grpc query is now deprecated, update to account-id(type of uint64) to use `AccountAddressByID`.
 * (x/group) [#13876](https://github.com/cosmos/cosmos-sdk/pull/13876) Fix group MinExecutionPeriod that is checked on execution now, instead of voting period end.
+* (store) [#]() simplify cachekv store with copy-on-write btree, it's consensus-breaking because iteration is done on isolated view now, while it's not the case before.
```

Comment:
```suggestion
* (store) [#14350](https://github.com/cosmos/cosmos-sdk/pull/14350) simplify cachekv store with copy-on-write btree, it's consensus-breaking because iteration is done on isolated view now, while it's not the case before.
```